configuration/configuration.go

```diff
@@ -211,26 +211,6 @@ type Configuration struct {
 
 	// Compatibility is used for configurations of working with older or deprecated features.
 	Compatibility struct {
```

Comment:
I would argue for removing the empty Compatibility block as well.

Reply:
I left it there since we might want it again in the future for whatever reason. Then at least the format stays the same and people don't add it back with a new name.

Reply:
Does it matter if it comes back with a new name?

Reply:
Probably not, but thought it's nicer if it's consistent. I'll remove it now though.